rules/ftw-config.conf

```diff
@@ -1,5 +1,10 @@
 # Overrides default SecResponseBodyMimeType in order to add application/json (httpbin response Content-Type)
 SecResponseBodyMimeType text/plain text/html text/xml application/json
+# crs-setup.conf.example defaults SecAction only for phase 1 and 2.
```

Comment:
To confirm, is this issue common to any FTW user not just this repo? I guess so, and it's sort of weird that coreruleset defines tests which require specific setup like this file, but doesn't actually provide that file itself 🤔

Reply:
I don't recall the need for this tweak running go-ftw against ModSecurity. I'm thinking about some differences in internal Coraza behaviours that do not log phase 4 into the error log by default 🤔.

Just to briefly sum up the problem: go-ftw was not able to pass response body matched rules because Coraza is not printing the error log about them. Just adding `log` to the default action at phase 4 is a workaround: 
```
SecDefaultAction "phase:3,log,auditlog,pass"
SecDefaultAction "phase:4,log,auditlog,pass"
```
Still, I was not expecting to have to be explicit about it to make the tests succeed. @fzipi do you have any hunch about it?